styles/Editor.css

```diff
@@ -939,6 +939,47 @@
   font-size: 75%;
 }
 
+.ProseMirror .table-footer {
+  font-size: 14px;
+}
+
+.ProseMirror .table-footer > div:first-child {
```

Comment:
Maybe here too? What do you think?

Reply:
Yeah, these divs represent `footnotes_element`, so I can use decorations to set the class `footnote-element` for all these nodes across the document.

Reply:
Done!